javascript/packages/orchestrator/src/orchestrator.ts

```diff
@@ -678,7 +681,7 @@ export async function start(
 
     const collatorPromiseGenerators = [];
     for (const parachain of networkSpec.parachains) {
-      if (!parachain.addToGenesis && parachain.registerPara) {
+      if (!parachain.addToGenesis && parachain.registerPara && !networkSpec.relaychain.chainSpecPath) {
```

Comment:
Could be the case that the `parachain` is not in the `chain-spec` and we want to register it. And in case that the chain-spec contains the parachain the user can set

```toml
add_to_genesis = false
register_para = false
````
 for the specific parachain.

Reply:
Ok I 

> Hi @samelamin, looks good. Just a couple of small comments. Can you also run:
> 
> ```shell
> npm run lint:write
> ```
> 
> Since the CI is failing because of the linter.
> 
> Thanks!!

Thanks @pepoviola I shall resolve these tomorrow morning and try out some of the changes you requested with `register_para` set to false

Reply:
Thanks @samelamin, ping me if you have questions.